README.md

```diff
@@ -1,24 +1,247 @@
+
+We're a group of 3 members: [@tobuya](https://github.com/tobuya), [@tanveerisonline](https://github.com/tanveerisonline), [@IndieCoderMM](https://github.com/indiecodermm)
+
+<img src="./kanban.png" width=600 height=400 alt="kanban board" />
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🧰 Tech Stack  <a name="tech-stack"></a>
+
+- **Back-end** 💻: 
+  - [Ruby on Rails](https://rubyonrails.org/) - Creating API endpoints and managing databases
+  - [PostgreSQL](https://www.postgresql.org/) - Database management system
+
+
+- **Front-end** 🎨: 
+  - [React](https://reactjs.org/) - Building user interfaces
+  - [Redux](https://redux.js.org/) - Managing states and API calls
+  - [Bootstrap](https://react-bootstrap.github.io/) - Pre-built UI components
+
+
+- **Tools** 🛠: 
+  - [Rspec](https://rspec.info/) - Testing models and requests
+  - [RSwag](https://github.com/rswag/rswag) - Documenting and testing API endpoints
+  - [Jest](https://jestjs.io/) - Unit tests for frontend components
+  - [Webpack](https://webpack.js.org/) - Module bundler for React/javascript
+  - [Render](https://render.com/) - Cloud platform for deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ✨ Key Features  <a name="key-features"></a>
+
+- Browse list of available courses
+- Log in with a username and access all features
+- Reserve online classes according to preferred time and location
+- View upcoming classes and reservations
+- Mobile-friendly and responsive design
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📘 Getting Started  <a name="getting-started"></a>
+
+To run this project locally, you'll need to follow these steps.
+
+### 📋 Prerequisites
+
+Make sure you have the following installed on your machine:
+- [Ruby 3.1.3 or higher](https://www.ruby-lang.org/en/)
+- [Rails 7.0.4 or higher](https://rubyonrails.org/)
+- [PostgreSQL 15.2 or higher](https://www.postgresql.org/)
+- [Node 18.14 or higher](https://nodejs.org/en)
+
+### 📂 Setup
+
+Clone this repository to your desired foler.
+
+```sh
+cd my-project
+git clone git@github.com:IndieCoderMM/e-learning-academy.git .
+```
+
+### 📥 Installation
+
+Install the required gems with:
+
+```sh
+bundle install
+```
+
+Install the node dependencies with:
+
+```sh
+npm install
+```
+
+### 💾 Database
+
+Create the databases and run migrations with:
+
+```sh
+rails db:create
+rails db:migrate
+```
+
+To load the sample data, run:
+
+```sh
+rails db:seed
+```
+
+### 💻 Usage
+
+Build the frontend assets with:
+
+```sh
+npm run build
+```
+
+To run the development server, execute the following command:
+
+```sh
+rails server
+```
+*Once the server is up and running, you can access the website on  `http://localhost:3000`*
+
+### 📃 API
+
+To read the API documentation, visit `https://localhost:3000/api-docs`
+
+### 🧪 Tests
+
+To run tests, run the following command:
+
+```sh
+rspec spec --format doc
+```
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. 

![image](https://github.com/IndieCoderMM/e-learning-academy/assets/57421823/f148dd01-0bd3-4c34-8351-168784d68583)


Kindly consider deleting that folder if you are not using it. 👍